src/themes/default/stylesheets/main.scss

```diff
@@ -387,7 +387,7 @@ $background-arguments: #fafbfc;
     .operation-description > h5,
     .definition-description > h5,
     .definition-properties > h5 {
-      display: none;
+      @include visually-hidden();
```

Comment:
Looks like this one is in the `spectaql` theme as well. Can you add that one and test it too?

Reply:
(meaning `src/themes/spectaql/*`)